lists/cm/cm-niu.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": {
+    "en": "Tax Identification Number (Cameroon)",
+    "local": "Numéro d'Identifiant fiscal Unique (NIU)"
+  },
+  "url": "https://www.impots.cm/",
+  "description": {
+    "en": "The Directorate General of Tax (Direction Générale des Impôts) of Cameroon issues a unique tax identification number (Numéro d'Identifiant fiscal Unique, or NIU) to all individuals and legal persons liable to pay tax in the country. "
+  },
+  "coverage": [
+    "CM"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "charity",
+    "company",
+    "trust",
+    "unincorporated_body"
+  ],
+  "sector": [],
+  "code": "CM-NIU",
+  "confirmed": true,
+  "deprecated": false,
+  "listType": "primary",
+  "access": {
+    "availableOnline": true,
+    "onlineAccessDetails": "",
+    "publicDatabase": "http://teledeclaration-dgi.cm/",
+    "guidanceOnLocatingIds": "At http://teledeclaration-dgi.cm/ use the link ' Vérifier un NIU' to check an NIU. Alternatively you can search via name or NIU here: http://teledeclaration-dgi.cm/modules/immatriculation/Consultation/listecontribuable.aspx.",
+    "exampleIdentifiers": "M099600009248R",
```

Comment:
I couldn't find this identifier by following the guidance. I could find a list of companies at http://teledeclaration-dgi.cm/modules/immatriculation/Consultation/listecontribuable.aspx by filtering on the "Centre de rattachement" drop down. I was then able to choose an ID (P017800471223D) and look up its entry through the interface.

Reply:
Hmmm. It works for me. If you go to http://teledeclaration-dgi.cm/modules/immatriculation/Consultation/listecontribuable.aspx and put 'plan international' in the 'nom' box or 'M099600009248R' in the NIU box then press 'Afficher' then it will bring up the entry.

Reply:
Yes, that definitely works. I don't know what I was doing wrong but PEBKAC alright.